Add Danish localization

c960657 opened the pull request:
Add translation for Danish (da).

Demo site: https://bsky.aggemam.dk

auroursa commented on src/screens/Messages/ChatList.tsx:
```diff
@@ -165,7 +165,9 @@ export function MessagesScreen({navigation, route}: Props) {
                       onPress={() => refetch()}>
-                      <ButtonText>Retry</ButtonText>
+                      <ButtonText>
+                        <Trans>Retry</Trans>
```
I think this should be reviewed separately as a PR, rather than included in the Danish localization.

c960657 replied:
Done: #7243

pfrazee commented:
Awesome, thank you @c960657 ! If you could get one other danish speaker to review this, I can get it merged!

simonschultz reviewed (commented):
Looking good !!!

jyskweb reviewed (commented):
Grammatical error in line [messages.po#L8266](https://github.com/bluesky-social/social-app/blob/6ecc568d44eb2b686bd8c70bc84a4cfc2a5b185c/src/locale/locales/da/messages.po#L8266)

jyskweb reviewed (commented):
Error in navbar.

![Screenshot_20250102_014704_Firefox](https://github.com/user-attachments/assets/dcd03ced-db2a-46dc-a076-a95e8b765adf)

auroursa commented on src/locale/locales/da/messages.po:
```diff
@@ -0,0 +1,8274 @@
+
+#: src/view/shell/Drawer.tsx:108
+msgid "<0>{0}</0> {1, plural, one {following} other {following}}"
+msgstr "<0>{0}</0> {1, plural, one {følger} other {følger}}\""
```
```suggestion
msgstr "<0>{0}</0> {1, plural, one {følger} other {følger}}"
```

auroursa reviewed (commented):
I think this causing the navbar issue.

c960657 commented:
Thanks for the reviews. The [stray quote in the navbar](https://github.com/bluesky-social/social-app/pull/7220/commits/4e168fa4e3f70b61012e6b4f5b39072782f2800b) and the [error in line 8266](https://github.com/bluesky-social/social-app/pull/7220/commits/dd0f5a02c23c6dcf614a9be90fc2d574454be04c) have been fixed now.

c960657 commented:
There is a suggestion here about translating “post” to “indlæg” (instead of just using the English term) and “repost” to “videredel”/“videredeling”. I tend to agree.

https://bsky.aggemam.dk/profile/esben.leonhard-frederiksen.dk/post/3leqwinep6224

jyskweb reviewed (commented):
Looks great 👍

renejsum commented on src/locale/locales/da/messages.po:
```diff
@@ -0,0 +1,8274 @@
+#: src/screens/Settings/Settings.tsx:215
+#: src/screens/Settings/Settings.tsx:218
+msgid "About"
+msgstr "Andet"
```
Om ?

c960657 replied:
Thanks. Seems like a brainfart.

surfdude29 commented on lingui.config.js:
```diff
@@ -1,6 +1,7 @@
 /** @type {import('@lingui/conf').LinguiConfig} */
 module.exports = {
   locales: [
+    'da',
```
Would it be possible to keep the list in alphabetical order and put `da` after `ca` please? 🙏 (I think `en` is listed first because it's the default and fallback locale.)

surfdude29 commented:
Also, if you could please add a `'da',` entry to `app.config.js` here, that would be great:
https://github.com/bluesky-social/social-app/blob/59709ad8d8743ae53272b1852b6948ea37eebfed/app.config.js#L77-L78

This will enable Danish to be displayed as a supported language in the App Store listing, and it will also allow the `TextInput` context menus to be translated into Danish for iOS users who have this set as their device language :)

kseistrup commented:
I've corrected a few typos. Please see attached diff:

:paperclip: [rettelser-diff.txt](https://github.com/user-attachments/files/18298362/rettelser-diff.txt)

Looks good otherwise.

emilstahl commented:
LGTM 👍

c960657 commented:
The dust has settled on this PR now, and I think it is ready to be merged.

I have addressed all issues raised here and in the Bluesky thread linked above, and I don't think there are any strong disagreements.

surfdude29 commented:
In case it's helpful to know, the new Danish localization can be tried out on the dev server at: [main.bsky.dev](https://main.bsky.dev/)

This is what the @bsky.app profile looks like for instance:

![IMG_0456](https://github.com/user-attachments/assets/973af32f-34e4-4161-ad89-a901083e9400)

If you see anything that should be changed before the 1.97 release is cut, a new PR can be submitted in the next 24 hours or so and then it should make it into the release, as described here:
- #7317
